.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build
+
+on:
```

Comment:
I think this should also run on pushes to the separate container branches.

However, this would generate all new containers for everything, not just the changed container.
It might take some tweaking.